packages/codegen-ui-react/lib/forms/form-renderer-helper/form-state.ts

```diff
@@ -30,9 +30,15 @@ import {
   PropertyAssignment,
   PropertyName,
 } from 'typescript';
+import { lowerCaseFirst } from '../../helpers';
 
 export const getCurrentValueName = (fieldName: string) => `current${capitalizeFirstLetter(fieldName)}Value`;
 
+export const getCurrentDisplayValueName = (fieldName: string) =>
+  `current${capitalizeFirstLetter(fieldName)}DisplayValue`;
+
+export const getRecordsName = (modelName: string) => `${lowerCaseFirst(modelName)}records`;
```

Comment:
my pr uses camel case (https://github.com/aws-amplify/amplify-codegen-ui/pull/722/files) so this would be Records instead of records to match golden file

Reply:
Done